src/schema/schema_create.c

```diff
@@ -205,13 +205,18 @@ __create_file(
     }
 
     /*
-     * If creating an ordinary file, append the file ID and current version numbers to the passed-in
-     * configuration and insert the resulting configuration into the metadata.
+     * When creating an ordinary file reconfigure parts of the metadata and insert the resulting
```

Comment:
I think we can clean up this comment. Consider something like "If creating an ordinary file, update the file ID and current version numbers and strip the incremental backup information and checkpoint LSN from the extracted metadata".

Reply:
done! 👍